frontend/src/styles/mixins.scss

```
@@ -0,0 +1,15 @@
+@mixin standard_button {
+  height: 50px;
+  width: 100%;
+  background: transparent;
+  border-radius: 8px;
+  border: 2px solid $contrast-color;
+  color: $contrast-color;
+  margin-top: 24px;
+  cursor: pointer;
+  font-size: 1.125rem;
+  -webkit-transition: 0.3s;
+  -moz-transition: 0.3s;
+  -o-transition: 0.3s;
+  transition: 0.3s;
```

You  can remove the prefixed attributes. The last version of any browser that needed it was released in 2014.
```suggestion
  transition: 0.3s;
```

---

✅